modelplugin/ModelGenerator.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+TYPEVERSION=TestDevice-2.0.0
```

Comment:
I would parametric this script in order for it to be used with any possibile given model/org

Reply:
Can I create an issue to do this as a follow up? I didn't want to make this patch too big

Reply:
+1 on issue for this